salt/modules/network.py

```diff
@@ -490,7 +490,7 @@ def _netstat_route_sunos():
             'gateway': comps[1],
             'netmask': '',
             'flags': comps[2],
-            'interface': comps[5]})
+            'interface': comps[5] if len(comps) == 6 else ''})
```

Comment:
Is it possible for this array to be longer than 6? I am wondering if `>=` might be a safer option here.